docs/dev/driver.rst

```diff
@@ -162,6 +162,27 @@ The driver object contains the following attributes:
       String containing the distribution ID of Firefox. This value is
       ``undefined`` on Firefox versions older than 48.0.
 
+.. js:function:: addons.get(addonId)
```

Comment:
I'm a little confused why the context doesn't contain the docs for install/uninstall. Are they documented somewhere else? Naively, I would expect them to be in the same place.